client/my-sites/checkout/composite-checkout/lib/existing-card-processor.ts

```diff
@@ -9,13 +9,33 @@ import type { PaymentProcessorResponse } from '@automattic/composite-checkout';
 /**
  * Internal dependencies
  */
-import { createTransactionEndpointRequestPayloadFromLineItems } from './translate-cart';
+import {
+	createTransactionEndpointRequestPayload,
+	createTransactionEndpointCartFromResponseCart,
+} from './translate-cart';
 import submitWpcomTransaction from './submit-wpcom-transaction';
 import type { PaymentProcessorOptions } from '../types/payment-processors';
-import type { ExistingCardTransactionRequestWithLineItems } from '../types/transaction-endpoint';
+import type { TransactionRequest } from '../types/transaction-endpoint';
 
 const debug = debugFactory( 'calypso:composite-checkout:payment-method-helpers' );
 
+type ExistingCardTransactionRequest = Omit<
```

Comment:
I think this might be a little easier to read if the Omit/Partial were moved together. Since there's already a Pick in the Required half, it wouldn't need to know about the Omit.